zephyr/module.yml

```
@@ -0,0 +1,5 @@
+---
+name: zephyr-config
```

This should be something better matching, e.g. 


```suggestion
name: zaphod-config
```

---

Oh my goodness! I must have gone a little snow-blind with all the Zs and PHs :)